discovery-provider/ddl/migrations/0000_schema_failable.sql

```diff
@@ -1166,9 +1166,9 @@ CREATE FUNCTION public.on_new_row() RETURNS trigger
 begin
   case TG_TABLE_NAME
     when 'tracks' then
-      PERFORM pg_notify(TG_TABLE_NAME, json_build_object('track_id', new.track_id)::text);
+      PERFORM pg_notify(TG_TABLE_NAME, json_build_object('track_id', new.track_id, 'updated_at', new.updated_at, 'created_at', new.created_at, 'blocknumber', new.blocknumber)::text);
     when 'users' then
-      PERFORM pg_notify(TG_TABLE_NAME, json_build_object('user_id', new.user_id)::text);
+      PERFORM pg_notify(TG_TABLE_NAME, json_build_object('user_id', new.user_id, 'blocknumber', new.blocknumber)::text);
```

Comment:
@stereosteve is this the right way to update this trigger? or do I need to make a new ddl file recreating it?

Reply:
you should do a new migration or function definition that does `CREATE OR REPLACE FUNCTION` probably

Reply:
cool thanks, was going off of this but maybe I misread
https://github.com/AudiusProject/audius-protocol/tree/main/discovery-provider/ddl#example-usage
>Any change to a file will cause the file to be re-run the next time you invoke bash pg_migrate.sh. Whitespace changes are ignored.

Reply:
It would probably skip b/c of this md5 stuff: https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/ddl/pg_migrate.sh#L44

Reply:
yes you should not directly edit `0000_schema_failable.sql`... it is meant to be a snapshot of the DB state when transitioning from alembic.

Adding a new migration file is the way to affect changes.

Reply:
And the triggers live in the `functions` dir... so you can make changes there to update trigger source.

Reply:
ohh I see... this is in the trigger that `es-indexer` creates... which predates pg_migrate... so we should probably move those into `functions` dir.

Reply:
for historical reference that was done here, thanks Steve! https://github.com/AudiusProject/audius-protocol/pull/6207